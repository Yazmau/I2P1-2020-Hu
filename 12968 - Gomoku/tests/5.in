30
...b...........
..........b....
.....b.b..b....
...............
...w..bb...b...
.........w.....
...............
......w........
...............
...............
b...w......w...
....w....w...w.
...............
....b..........
..w.....w......
...............
...............
...............
.......b...w...
...............
........www....
....b...wbw....
........wbw....
.....w...b.....
.........b.....
.........b.....
.........w.....
...............
...............
...........bbbb
...............
...............
b..............
......b........
...............
...............
............w..
.bbb...........
.bwwwwwb.......
.bbb...........
...............
...............
...............
...............
...........wwww
...............
...www.........
...wbw.........
...wwb.........
......b........
.......b.......
........b......
.........wb...b
...............
...............
...............
.......w.......
...............
...............
.........w.bbbb
........bb.....
.....bb.......b
........b......
.......w.......
..bb....w......
...............
w..........w...
..w............
...............
............w..
...............
......w....w...
..bb...........
.w.............
.............wb
.........www...
.........wbw...
.........wbw...
..........b..b.
..........b....
..........b....
.b........w....
...............
..........w....
...............
......w........
...............
...............
...............
...........bbbb
..www..........
..wbbbbbw......
..www..b.......
.......w.......
...............
........b......
....w..........
...............
...............
...............
...............
...............
...............
...............
...........bbbb
...bbb.........
...bwbb........
...bbw.........
......w........
.......w..w....
........w......
.........b.....
........b......
...............
...............
...............
...............
...w...........
...............
............www
....b..........
...b.b.........
...............
.........w.....
.b.............
......b.....w.w
.bb..b.........
........b...b..
...............
..............w
.w........w....
...............
.......w...w...
...............
.w.........w...
...............
...............
...............
...............
...............
..b.......www..
..........wbw..
..........wbw..
..w..b.....b...
...........b...
...........b...
...........w...
...............
...............
.........w.bbbb
...............
...............
.b.............
...............
...............
...............
...b...........
............w..
...............
...............
.....w.........
......www......
......wbbbbbw..
......www......
...........bbbb
...............
...............
...............
...www.........
...wbw.b.......
...wwb.........
......b........
.......b.......
..b.....b......
.........w...w.
...............
...............
...............
...............
.......w...bbbb
...............
..........b....
.b........b....
.....b.b.......
...............
..........b....
...b.......b...
...w...........
......b........
.....w.b.w.....
..w............
.......www.....
...............
.........w.....
.w.......w.....
............b..
........b......
...............
...............
...............
...............
.www..w........
.wbw...........
wwbw...........
..b............
..b............
..b............
..w............
...............
...........bbbb
...............
...............
.....bb........
...............
...............
...bbb.........
...bwwwwwb.....
...bbb.........
...............
...............
............w..
............w..
...............
...............
............www
...............
...............
...............
...............
...............
...............
........www.b..
........wbw....
........wwb....
...........b...
............b..
..w..........b.
.....b........w
....w..........
...........bbbb
............b..
......b........
....b..........
.............b.
...............
b......w.......
.........b.....
.b..w..........
....w...w......
......w...bb...
........w......
.......b.......
.......w......w
...w...........
.............w.
...............
...............
..b............
........www....
........wbw....
..b.....wbw....
.........b.....
.........b.....
.........b.....
.........w.....
..............w
...............
...w...........
...............
...........bbbb
...............
...............
..............b
...............
b..............
...............
...............
...............
...............
......bbb......
....w.bwwwwwb.w
......bbb......
...............
...............
............www
...............
...............
...............
.........b.....
...............
...............
.....bbb....b..
.....bwb..w....
.....bbw.......
........w......
...w.....w.....
..........w....
...........b...
...............
............www
...........b...
...............
...b....b......
...b...........
........b....b.
........w......
............b..
b..........w...
....wb.........
......w........
.w.............
....b..........
w.w........w...
....w.w........
..............b
...............
....b..........
...............
...............
...............
..bbb..........
..bwb..........
..bwb..........
...w...........
...w...........
...w.....b.....
...b...w.......
...............
.............w.
............www
...............
...............
...............
......www......
......wbbbbbw..
....b.www......
...............
...........b...
.........w.....
...............
...............
...............
...............
...............
..........wbbbb
...............
...............
.......b.......
...............
...............
...........b...
.......w.......
...............
.bbb...........
.bwb...........
.bbw...........
....w..........
.....w.........
.....ww........
.......b....www
...............
.........b.....
.b...b.........
...............
..........w....
..........b....
.b.............
.....w........w
....b...b......
.w.............
...............
...w....wb.....
b..............
b.w...w........
........ww.....
.............b.
.......www..b..
.......wbw.....
.......wbw.....
........b......
........b......
........b......
.....w..w......
...............
...............
...............
...............
...............
...............
.w.........bbbb
...............
...www.........
...wbbbbbw.....
...www.........
............b..
...............
...............
...b...........
...............
...............
...............
...w...........
...............
...............
.w.........bbbb
...............
...............
...............
........www....
........wbw.b..
......w.wwb....
...........b...
............b..
.............b.
..............w
...............
...............
...............
...............
..b..w.....bbbb
..b.........b..
.b.............
...............
..b.b...b......
..b............
b..............
....w..........
.............b.
...w...w.......
..........w....
...............
........b....w.
...........ww..
.............w.
w...w.........b
...............
...............
...............
..........b....
.......b.......
...............
.......www.....
ww.....wbw.....
.......wbw.....
........b......
........b......
........b......
........w......
...............
...........bbbb
