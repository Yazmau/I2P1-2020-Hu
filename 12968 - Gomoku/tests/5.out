none
black
white
black
none
black
black
white
none
black
black
black
none
black
white
black
none
black
white
white
none
white
black
white
none
black
black
black
none
black
